lib/cloudant.js

```diff
@@ -712,7 +706,10 @@ Cloudant.prototype.updateAttributes = function(model, id, data, options, cb) {
     _.mergeWith(doc, data, function(dest, src) { return src; });
     self.create(model, doc, options, function(err) {
       if (err) return cb(err);
-      return cb(err, data);
+      mo.db.get(id, function(err, doc) {
```

Comment:
That's the issue me and @jannyHou were just talking about. `_insert` returns back only the `_rev` and `id` property and not the whole instance.

Why it does that? Because `.create()` in dao stores the to be inserted data and once the call from `_insert` is returned, it just appends the `id` and `_rev` value back to the stored data and sends it back to the user.

This makes using `_insert` function not so useful because one might think they are getting back the whole instance that got inserted but instead they get back the `id` and `_rev`.